packages/block-editor-tools/.eslintrc.json

```diff
@@ -2,8 +2,6 @@
   "extends": ["@alleyinteractive/eslint-config/typescript-react"],
   "rules": {
     "import/no-cycle": "off",
-    "react/jsx-props-no-spreading": "off",
-    "react/react-in-jsx-scope": "off",
```

Comment:
These rules are located in the `@alleyinteractive/eslint-config/typescript-react` package and not needed here since it is extended.